C-Plus-Plus/math/Unique_Number.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+
+You are given an integer array nums. The unique elements of an array are 
+the elements that appears exactly once in the array. Return the sum of all 
+Unique elements of nums.
+
+INPUT : [1, 5, 2, 6, 5]
+OUTPUT : 9
+
+Explanation : Unique elements are [1, 2, 6] and the Sum is 9
+
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main(){
```

Comment:
Don't put your code logic in `main()` function. Use `main()` for the driver code